lunch-go-where/src/pages/Home.js

```diff
@@ -7,55 +7,123 @@ import apis from "../utils/apiCalls";
 import IndividualCard from "../components/IndividualCard";
 
 const Home = () => {
-	const [ location, setLocation ] = useState("");
-	const [ recStalls, setRecStalls ] = useState([]);
-	const handleLocationChange = (evt, value) => {
-		setLocation(value);
-	};
-	useEffect(() => {
-		apis
-			.getRecommendedStalls()
-			.then((data) => {
-				setRecStalls(data);
-				// console.log('RETURNING ', data);
-			})
-			.catch((err) => console.error(err));
-	}, []);
-	return (
-		<React.Fragment>
-			<ResponsiveAppBar />
-			<Typography variant="h4">SEARCH</Typography>
-			<Box>
-				<form>
-					<AutocompleteLocation handleFieldChange={handleLocationChange} />
-					<Slider label={"Price Range"} step={5} defaultValue={5} min={0} max={20} />
-					<Slider label={"Wait Time"} step={5} defaultValue={5} min={0} max={30} />
-					<Button color="secondary" type="submit" variant="contained">
-						SEARCH
-					</Button>
-				</form>
-			</Box>
-			<Box sx={{ display: "flex", justifyContent: "space-evenly" }}>
-				{recStalls ? (
-					recStalls.map((stall) => {
-						console.log(stall.stallName);
-						return (
-							<React.Fragment>
-								<IndividualCard
-									// img={stall.img}
-									id={stall._id}
-									nameOfStall={stall.stallName}
-									cuisine={stall.cuisine}
-									location={stall.location}
-								/>
-								{/* <Typography>Submitted by: {stall.author.name}</Typography> */}
-							</React.Fragment>
-						);
-					})
-				) : null}
-			</Box>
-		</React.Fragment>
-	);
+    const [location, setLocation] = useState("");
+    const [recStalls, setRecStalls] = useState([]);
+    const [priceRange, setPriceRange] = useState("5");
```

Comment:
Can we set the state for both price and wait time as a number, not a string, since we need a number to evaluate the results on the backend? Usually we set `value` of the component to be equal to `state`. 

So we can just shift the `parseInt` function into the `handlePriceChange` and `handleTimeChange` functions, e.g. `setPriceRange(parseInt(evt.target.value))`. Then in the Slider, `value={priceRange}`. See if that looks more intuitive?